Assets/Plugins/Source/Editor/Configs/SteamConfig.cs

```diff
@@ -25,23 +25,61 @@
 
 namespace PlayEveryWare.EpicOnlineServices
 {
+    using Editor;
+    using Editor.Config;
+    using Editor.Utility;
+
     [Serializable]
+    [ConfigGroup("Steam Configuration")]
+    // TODO: Make SteamConfig derive from EditorConfig, and update the native code
+    //       to properly reference the correct file where appropriate.
     public class SteamConfig : Config
     {
+        [ConfigField("Steam Flags", ConfigFieldType.TextList)]
+        [Obsolete] // This is marked as obsolete because it is actually not utilized anywhere.
```

Comment:
I want to huddle about this and validate it.

Reply:
In light of the fact that this is utilized in the native code, it ought _not_ be marked obsolete. It's possible that this is an indication that this is broken to some extent.

Reply:
Removed the `[Obsolete]` attribute. Marking this conversation as resolved.